targets/xorg

```diff
@@ -102,16 +163,43 @@ END
     # dependency of ubuntu-minimal-1.267 (precise), which we did not install.
     # Looks like an apt bug, I guess we can live with that.
     apt-get -y --force-yes --no-install-recommends dist-upgrade -f
+elif [ -f "$pinfile" -o -f "$pinlist" ]; then
+    # No longer need the pin; remove it
+    rm -f "$pinfile" "$pinlist"
+    apt-get update || true
+    dpkg -r --force-depends "$pindummypkg"
+    apt-get -y --force-yes --no-install-recommends dist-upgrade -f
 fi
 
-install xorg xserver-xorg-video-fbdev$ltspackages
-if [ "${ARCH#arm}" = "$ARCH" ]; then
-    install xserver-xorg-video-intel$ltspackages
+# Install backported xorg instead of the default one
+if [ -n "$backport" ]; then
+    # We still install xorg later to pull in its dependencies
+    install --minimal "xserver-xorg$backport" "libgl1-mesa-glx$backport" \
+        "libegl1-mesa$backport" "libgles2-mesa$backport" \
+        "xserver-xorg-input-synaptics$backport" $installvideodrivers
+fi
+
+# Install xorg, no video drivers
+if [ -z "$inteldriver" -a -n "$backport" ] && release -eq precise; then
+    # xorg is packaged wrong on ARM and conflicts with xserver-xorg-lts-quantal
+    # so replace it with something not broken
+    install_dummy xorg -- xserver-xorg$backport libgl1-mesa-glx$backport \
+        libgl1-mesa-dri$backport libglu1-mesa xfonts-base x11-apps \
+        x11-session-utils x11-utils x11-xfs-utils x11-xkb-utils \
+        x11-xserver-utils xauth xinit xfonts-utils xkb-data xorg-docs-core \
+        xterm x11-common xinput
```

Comment:
Because that's a list of requirements for the dummy xorg package, not an actual install line.  The install line in that case happens above (`[ -n "$backport" ]`).